src/main/resources/scripts/dhis2_rights.sql

```diff
@@ -0,0 +1,2 @@
+INSERT INTO referencedata.rights (id, description, name, type) VALUES ('85abf93a-ea22-42a9-be7c-5baf64d757f9', NULL, 'DHIS2_ADMIN', 'GENERAL_ADMIN');
```

Comment:
change this script to ssql migration, it should be executed automatically on the server

Reply:
Unluckily when migration is being applied, it does not have an access to tables other than dhis2. I'm afraid it is not possible to be automated using current setup.
![image](https://github.com/villagereach/selv-v3-dhis2-integration/assets/113436691/571a3697-171c-40fd-a00e-aa7a3a5c085a)